src/patternfly/components/Button/button.scss

```
@@ -705,6 +707,12 @@
       }
     }
   }
+
+  &.pf-m-notify .#{$button}__icon {
+    animation-name: #{$button}-icon-notify;
+    animation-duration: var(--#{$button}__icon--AnimationDuration--notify);
+    animation-timing-function: var(--#{$button}__icon--AnimationTimingFunction--notify);
```

If you made the other suggestion, we'd also want to update this

```suggestion
    animation-duration: var(--#{$button}--m-notify__icon--AnimationDuration--notify);
    animation-timing-function: var(--#{$button}--m-notify__icon--AnimationTimingFunction--notify);
```